frontend/server/src/Validators.php

```diff
@@ -158,6 +158,45 @@ public static function alias(string $alias): bool {
         );
     }
 
+    /**
+     * Returns whether the username or email is valid.
+     *
+     * @param string $usernameOrEmail
+     * @return boolean
+     */
+    public static function username(string $usernameOrEmail): bool {
```

Comment:
```suggestion
    public static function usernameOrEmail(string $usernameOrEmail): bool {
```

hay que ser explícitos con qué se está validando.

Reply:
Listo